src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -215,6 +219,82 @@ private static void addSecurityConfigs(ConfigDef configDef) {
     );
   }
 
+  private static void addConnectorTimeoutConfigs(ConfigDef configDef) {
```

Comment:
you can actually just add an exception in `checkstyle/suppressions.xml`. See my comment #384 for more details.

Reply:
I have added the exception for file ElasticsearchSinkConnectorConfig.java and it build fine now.
Thanks!